internal/factsengine/gatherers/saphostctrl.go

```diff
@@ -0,0 +1,142 @@
+package gatherers
+
+import (
+	"regexp"
+	"strings"
+
+	log "github.com/sirupsen/logrus"
+	"github.com/trento-project/agent/pkg/factsengine/entities"
+	"github.com/trento-project/agent/pkg/utils"
+)
+
+const (
+	SapHostCtrlGathererName               = "saphostctrl"
+	saphostCtrlListInstancesParsingRegexp = `^\s+Inst Info\s*:\s*([^-]+?)\s*-\s*(\d+)\s*-\s*([^,]+?)\s*-\s*(\d+),\s*patch\s*(\d+),\s*changelist\s*(\d+)$`
+	saphostCtrlPingParsingRegexp          = `(SUCCESS) \( *(\d+) usec\)`
+)
+
+// nolint:gochecknoglobals
+var (
+	SapHostCtrlCommandError = entities.FactGatheringError{
+		Type:    "saphostctrl-cmd-error",
+		Message: "error executing saphostctrl command",
+	}
+	SapHostCtrlUnsupportedFunction = entities.FactGatheringError{
+		Type:    "saphostctrl-func-error",
+		Message: "requested function not whitelisted",
```

Comment:
They are call `webmethods`. Maybe we should use this term instead of function.
What do you think?